src/index.ts

```diff
@@ -274,7 +274,6 @@ async function update() {
     try {
       await autoUpdater.checkForUpdates();
     } catch (error) {
```

Comment:
Don't we want to do anything with the error?

Reply:
Nothing to do. Besides, it mostly happens when the customer is running us from a temp location instead of installing us.